src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -112,25 +118,62 @@ export class HaAutomationEditor extends KeyboardShortcutMixin(LitElement) {
         .backCallback=${this._backTapped}
         .tabs=${configSections.automations}
       >
-        <ha-button-menu
-          corner="BOTTOM_START"
-          slot="toolbar-icon"
-          @action=${this._handleMenuAction}
-          activatable
-        >
+        <ha-button-menu corner="BOTTOM_START" slot="toolbar-icon">
           <ha-icon-button
             slot="trigger"
             .label=${this.hass.localize("ui.common.menu")}
             .path=${mdiDotsVertical}
           ></ha-icon-button>
 
           <mwc-list-item
-            aria-label=${this.hass.localize(
-              "ui.panel.config.automation.editor.edit_ui"
-            )}
             graphic="icon"
-            ?activated=${this._mode === "gui"}
+            .disabled=${!stateObj}
+            @click=${this._runActions}
+          >
+            ${this.hass.localize("ui.panel.config.automation.editor.run")}
+            <ha-svg-icon slot="graphic" .path=${mdiPlay}></ha-svg-icon>
+          </mwc-list-item>
+
+          ${stateObj
+            ? html`<a
+                href="/config/automation/trace/${this._config
+                  ? this._config.id
+                  : ""}"
+                target="_blank"
+                .disabled=${!stateObj}
+              >
+                <mwc-list-item graphic="icon" .disabled=${!stateObj}>
+                  ${this.hass.localize(
+                    "ui.panel.config.automation.editor.show_trace"
+                  )}
+                  <ha-svg-icon
+                    slot="graphic"
+                    .path=${mdiTransitConnection}
+                  ></ha-svg-icon>
+                </mwc-list-item>
+              </a>`
+            : ""}
+
+          <mwc-list-item graphic="icon" @click=${this._promptAutomationAlias}>
```

Comment:
Let's put rename on top, I think it will be the most popular option.

Reply:
Hmz yeah maybe, but once renamed it probably won't happen much more often.
Runs are done over time more often for that matter.

Anyways, I ordered it to be consistent with the automation actions and conditions overflow menus:

![CleanShot 2022-08-25 at 21 19 35](https://user-images.githubusercontent.com/195327/186750397-1cd57194-4302-4ac4-bdcf-5e085f42e214.png)

Which have their testing/run capabilities listed first too.